buildconfig/Jenkins/buildscript

```diff
@@ -11,6 +11,19 @@
 ###############################################################################
 SCRIPT_DIR=$(dirname "$0")
 
+###############################################################################
+# Terminate existing Xvfb sessions
+###############################################################################
+if [[ -n ${CLEAN_XVFB} && "${CLEAN_XVFB}" == true ]]; then
```

Comment:
`&& $(command -v xvfb-run)` should be here too. Don't bother if the tool isn't installed.